src/main.cpp

```diff
@@ -35,7 +35,7 @@
 
 int main(int argc, char **argv){
 
-    ros::init(argc, argv, "hero_simulator");
+    ros::init(argc, argv, "pyro_simulator");
```

Comment:
Best to set this based on `use_pyro`.  As the node can be initialized after reading the config.